cpp/ql/lib/semmle/code/cpp/XML.qll

```diff
@@ -174,8 +177,8 @@ class XmlDTD extends XmlLocatable, @xmldtd {
   }
 }
 
-/** DEPRECATED: Alias for XmlDTD */
-deprecated class XMLDTD = XmlDTD;
+/** DEPRECATED: Alias for XmlDtd */
+deprecated class XMLDTD = XmlDtd;
```

Comment:
I take it we haven't released with `XmlDTD`?  So we don't also need
```
deprecated class XmlDTD= XmlDtd;
```
as well?

Reply:
Correct, we don't need that.   
I've removed the redundant deprecation in [this commit](https://github.com/github/codeql/commit/82d9180892314c87e6e778f209edea1c26efcd59).